bilidan.py

```diff
@@ -300,13 +300,18 @@ def launch_player(video_metadata, media_urls, comment_out, is_playlist=False, in
         video_size = (1920, 1080)
 
     logging.info('Loading comments...')
-    comment_out = convert_comments(video_metadata['cid'], video_size)
+    if comment is None:
+        comment_out = convert_comments(video_metadata['cid'], video_size)
+    else:
+        comment_out = open(comment, 'r')
+        comment_out.close()
```

Comment:
Because convert_comments closed before return, and only its path is actually used.